docs/docs/gatsby-style-guide.md

```diff
@@ -233,22 +210,34 @@ Here are specific formatting guidelines for any code:
   }
   ```
 
-# Adding images to articles
+## Format language keywords as code
 
-For including images, if the images aren't already hosted somewhere else on the
-web, you'll need to put them online yourself. A good way to do this is to commit
-them to a GitHub repository of your own, then push them to GitHub. Then you can
-right click the image and copy its image source.
+Format language keywords as code - this is done with the backtick key (located
+to the left of the "1" key on a US keyboard) in GitHub-flavored markdown. For
+example, put backticks around HTML tag names or CSS property names.
 
-Then you'd just need to reference them in your markdown file with this syntax:
-`[your alt text](your url)`
+## Use clear hyperlinks
 
-Then the images should show up when you click the "preview table" tab.
+Hyperlinks should contain the clearest words to indicate where the link will lead you. So instead of linking to the word [here](https://www.gatsbyjs.org/) link to [Gatsby's docs](https://www.gatsbyjs.org/).
+
+In tutorials that are meant for beginners, use as few hyperlinks as possible to minimize distractions. In docs, it's ok to include as many hyperlinks as necessary to provide relevant and interesting information and resources.
+
+## Begin sentences clearly
```

Comment:
Perhaps "indicate optionality" as this is what we're doing?

Reply:
I'll be honest: I have no idea what "indicate optionality" means.

Can we rephrase that to be friendlier to English as a second language/folks like me who don't read good? 😅

Reply:
"Clarify when a section is not required"

Reply:
;)